1_html-css/skills.html

```diff
@@ -0,0 +1,39 @@
+<html>
+  <html lang="en">
+    <head>
+      <meta charset="UTF-8" />
+      <meta name="viewport" content="width=device-width" />
+      <link rel="stylesheet" href="style.css" />
+      <title>Best website ever</title>
+    </head>
+    <body>
+      <div class="menu flex">
+        <div>
+          <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="contact.html">Contact</a></li>
+            <li><a href="skills.html">Skills</a></li>
+            <li><a href="portfolio.html">Portfolio</a></li>
+          </ul>
+        </div>
+        <div>
+          <img src="./linkedin-icon.png" />
+        </div>
+      </div>
+      <div class="name"></div>
+
+      <div class="begginer">
+        <h1>HTML begginer</h1>
```

Comment:
According to accessibility rules, you should have only one h1 per page. That is also better for optimizing a page for SEO - it will end up higher in Google search if you stick to a set of rules. All pages are eventually read by search engines crawlers - automatic readers that go through the code and index what's in it. It's important to follow the SEO rules if you're making a web shop for example - you want it to end up high in search results. That is again not that important for the exercise purpose but good to keep in mind that something like that exists.